### PR TITLE
Lakes

### PR DESCRIPTION
**Pull request by miriamseely**
Here is the metadata for the Lakes NHD layer.

### CODE REVIEW

File: metadata/water/lakes_nhd [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+## Data Page Link
+
+[https://gis.utah.gov/products/sgid/water/nhd-lakes/]
+
+## Update
+
+### Update Schedule
+
+<!--- The SGID Index does not specify how often this layer is updated. Moreover, the USGS website says that the NHD was retired in 2023 and will no longer be maintained. It says that the 3D Hydrography Program will contain the most recent data. Is this something we want to include in our metadata?
```
Comment (steveoh) [inline metadata/water/lakes_nhd:81]: **question**: as far as i know zach updated it as needed. I'm not sure I was aware that it was retired in 2023 and we should absolutely be migrating to the newest data. @ZachBeck and @gregbunce is this something you are aware of and planning for?

File: metadata/water/lakes_nhd [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+b0a55056-bdb9-4b7c-8812-8f246b328f88
+
+## Brief Summary
+
+Statewide bodies of water such as lakes, reservoirs, and rivers derived from the National Hydrologic Dataset (NHD).
+
+## Summary
+
+This dataset comprises bodies of water found in and around the state of Utah. This dataset is derived from the [National Hydrography Database](https://www.usgs.gov/national-hydrography/national-hydrography-dataset) (NHD) compiled by the United State Geological Survey [USGS](https://www.usgs.gov/) in partnership with UGRC and other state GIS programs.
```
[the remark this answers is not in the document]
Comment (miriamseely) [reply metadata/water/lakes_nhd:15]: The original metadata said we partnered with them, but I think it makes more sense to say that we modified and enhanced the data. Unless we really do partner with them-- @ZachBeck correct me if I'm wrong!

File: metadata/water/lakes_nhd [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+### How was the dataset created?
+
+The NHD was first created in the late 1990s by USGS in collaboration with the Environmental Protection Agency [(EPA)](https://www.epa.gov/). The original product, the National Hydrography Dataset, was created at a 1:100,000 scale for the conterminous United States. The EPA became the primary custodian of this medium resolution dataset in the early 2000s, while USGS and United States Forest Service [(USFS)](https://www.fs.usda.gov/) began the production of what would become the high resolution National Hydrography Database. Today, the high-resolution NHD is available at a nationwide 1:24,000 scale.
+
+More information on the history of this database can be found in [publications](https://www.horizon-systems.com/NHDPlusData/NHDPlusV21/Documentation/History/Making_the_Digital_Water_Flow.pdf) from USGS and on the [NHD website](https://www.usgs.gov/national-hydrography/national-hydrography-dataset#:~:text=In%20the%20late%201990s%2C%20the,and%20those%20of%20other%20medium).
```
[the remark this answers is not in the document]
Comment (miriamseely) [reply metadata/water/lakes_nhd:39]: That's a good point. I usually include schema information in What The Data Represents, but I see what you're saying about the additional fields being a part of how the dataset was created. I'll move things around.

File: metadata/water/lakes_nhd.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+Utah Lakes NHD
+
+## ID
+
+b0a55056-bdb9-4b7c-8812-8f246b328f88
+
+## Brief Summary
+
+Statewide areal bodies of water such as lakes, reservoirs, and ponds derived from the National Hydrography Dataset (NHD).
```
Comment (jacobdadams) [inline metadata/water/lakes_nhd.md:11]:
```suggestion
Statewide bodies of water including lakes, reservoirs, and ponds derived from the National Hydrography Dataset (NHD).
```

File: metadata/water/lakes_nhd.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+## Description
+
+### What is the dataset?
+
+The NHD is a national framework for assigning [reach addresses](https://enviro.epa.gov/enviro/ef_metadata_html.tri_page?p_column_name=reach_code#:~:text=Description%3A%20A%20reach%20code%20is,National%20Hydrography%20Dataset%20(NHD).) to water-related entities, such as industrial discharges, drinking water supplies, and fish habitat areas. Reach addresses establish the locations of these entities relative to one another within the NHD surface water drainage network, much like addresses on streets.
+
+The NHD contains reach codes for networked features, flow direction, names, and centerline representations for areal water bodies. The NHD also incorporates the National Spatial Data Infrastructure framework criteria established by the Federal Geographic Data Committee.
```
Comment (jacobdadams) [inline metadata/water/lakes_nhd.md:23]:
```suggestion
The NHD contains reach codes for networked features, flow direction, names, and centerline representations for water bodies. The NHD also incorporates the National Spatial Data Infrastructure framework criteria established by the Federal Geographic Data Committee.
```

File: metadata/water/lakes_nhd.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+### What does the dataset represent?
+
+This dataset approximates bodies of water as polygons that include attributes such as the area in square kilometers and [GNIS](https://www.usgs.gov/tools/geographic-names-information-system-gnis) name. For a full description of the fields in this dataset, please see the [schema summary](https://docs.google.com/document/d/1QWZKKPe3PWR9XyMYEG6qFmhsK55oBYYLWRQtknbrnD4/edit?usp=sharing) document.
+
+### How was the dataset created?
+
+The NHD was first created in the late 1990s by USGS in collaboration with the Environmental Protection Agency [(EPA)](https://www.epa.gov/). The original product, the National Hydrography Dataset, was created at a 1:100,000 scale for the conterminous United States. The EPA became the primary custodian of this medium resolution dataset in the early 2000s, while USGS and United States Forest Service [(USFS)](https://www.fs.usda.gov/) began the production of what would become the high resolution National Hydrography Database. Today, the high-resolution NHD is available at a nationwide 1:24,000 scale.
```
Comment (jacobdadams) [inline metadata/water/lakes_nhd.md:35]:
```suggestion
The current high-resolution, 1:24,000 scale NHD is an improvement of the original 1:100,000 scale dataset first created in the late 1990s by the USGS in collaboration with the Environmental Protection Agency [(EPA)](https://www.epa.gov/). It was created by the USGS and United States Forest Service [(USFS)](https://www.fs.usda.gov/) in the 2000s while the EPA maintained the original, moderate-scale dataset.
```

Trying to lead the paragraph with the most up-to-date info, but not sure how well I did.

File: metadata/water/lakes_nhd.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+More information on the history of this database can be found in [publications](https://www.horizon-systems.com/NHDPlusData/NHDPlusV21/Documentation/History/Making_the_Digital_Water_Flow.pdf) from USGS and on the [NHD website](https://www.usgs.gov/national-hydrography/national-hydrography-dataset#:~:text=In%20the%20late%201990s%2C%20the,and%20those%20of%20other%20medium).
+
+UGRC has modified the source NHD data to produce this layer. Four fields have been added to this feature class (FType_Text, FCode_Text, InUtah and IsMajor) and features have been split at the state boundary.  The "IsMajor" field identifies major lakes for coarse scale mapping and display, where 1 = major and 0 = minor. The "InUtah" field identifies whether a given body of water is within the state boundary, where 1 = in Utah and 0 = outside of Utah.
```
Comment (jacobdadams) [inline metadata/water/lakes_nhd.md:39]:
```suggestion
UGRC has modified the source NHD data by adding four fields (FType_Text, FCode_Text, InUtah and IsMajor) and splitting features at the state boundary.  The "IsMajor" field identifies major lakes for coarse scale mapping and display, where 1 = major and 0 = minor. The "InUtah" field identifies whether a given body of water is within the state boundary, where 1 = in Utah and 0 = outside of Utah.
```

File: metadata/water/lakes_nhd.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+## Summary
+
+This dataset comprises bodies of water found in and around the state of Utah. This dataset is derived from the [National Hydrography Dataset](https://www.usgs.gov/national-hydrography/national-hydrography-dataset) (NHD) compiled by the United State Geological Survey [USGS](https://www.usgs.gov/) and modified by UGRC to provide additional functionality for Utah-based users.
+
+## Description
+
+### What is the dataset?
+
+The NHD is a national framework for assigning [reach addresses](https://enviro.epa.gov/enviro/ef_metadata_html.tri_page?p_column_name=reach_code#:~:text=Description%3A%20A%20reach%20code%20is,National%20Hydrography%20Dataset%20(NHD).) to water-related entities, such as industrial discharges, drinking water supplies, and fish habitat areas. Reach addresses establish the locations of these entities relative to one another within the NHD surface water drainage network, much like addresses on streets.
```
Comment (jacobdadams) [inline metadata/water/lakes_nhd.md:21]: This feels like a lot of text to establish reach codes to then finally establish what the polygons are. Is there a way we could invert this, so we lead off with what the polygons are and then talk about reach codes at the end if it's necessary?

**Review by jacobdadams — CHANGES_REQUESTED** [review]
In general, most of these sections feel a little "upside down," where they spend the first few sentences establishing background info before finally mentioning the lakes layer later on. Can we reword it to be more upfront/direct? I've tried to make a couple suggestions, but I'm not sure I nailed them.

File: metadata/water/lakes_nhd.md (hunk 8 above)
Comment (miriamseely) [reply metadata/water/lakes_nhd.md]: I've moved some things around, let me know what you think! I'll probably keep fiddling with it as I work on Streams as well (since their metadata are so similar).

**Review by jacobdadams — APPROVED** [review]

File: metadata/water/lakes_nhd.md (hunk 8 above)
Comment (jacobdadams) [reply metadata/water/lakes_nhd.md]: It definitely reads better now. Thank you!
